UIKitHometasks/UIKitHometasks/EnterViewController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  EnterViewController.swift
+//  UIKitHometasks
+//
+//  Created by Валентин Коскин on 04.10.2022.
+//
+
+import UIKit
+/// экран входа
+final class EnterViewController: UIViewController {
+    @IBOutlet weak var loginTextField: UITextField!
```

Comment:
```suggestion
    @IBOutlet private weak var loginTextField: UITextField!
```
private добавь везде